### ISSUE TITLE
Fix windows behaviour in tests

### ISSUE THREAD

**Issue by eddelbuettel** [eddelbuettel/nanotime#28]
Dang. win-builder and R Hub fall over:

```
     Running 'data.frame.R'
     Running 'runTests.R'
   Warning message:
   running command '"C:/PROGRA~1/R/R-devel/bin/x64/R" CMD BATCH --vanilla  "runTests.R" "runTests.Rout"' had status 1 
    ERROR
   Running the tests in 'tests/runTests.R' failed.
   Last 13 lines of output:
         still.identical
     
     The following objects are masked from 'package:base':
     
         %in%, :, is.double, match, order, rank
     
     Error in runTestSuite(ts, useOwnErrorHandler = useOwnErrorHandler, verbose = verbose) : 
       invalid test suite supplied.
     Calls: doTest -> runTestFile -> runTestSuite
     In addition: Warning messages:
     1: In isValidTestSuite(testSuites) :
       specified directory ../nanotime/unitTests not found.
     2: In isValidTestSuite(testSuites[[1]]) :
       'testSuite' object is not of class 'RUnitTestSuite'.
     Execution halted
```

**Comment by eddelbuettel** [comment]
Better now a few commits in, last one being https://github.com/eddelbuettel/nanotime/commit/8d26713808b0f226863ac479b2eaaf33b548990a

@lsilvest : I disabled a metric ton of tests on Windows.  We may want to think about whether this is good enough for a release ("sure, who cares about Windows ...") or whether we want to review.  In most cases it actually points to RcppCCTZ.  

Working on this was brutal. Something wrong with the VM I used.  So slooooooooooooooooow...
